jvm/jcstress/PURPOSE

```diff
@@ -6,7 +6,3 @@ The Java Concurrency Stress tests (jcstress) is an experimental harness and a su
 to aid the research in the correctness of concurrency support in the JVM, class libraries,
```

Comment:
Same problem here in the initial paragraph.

Reply:
Should be fixed now.